test/net/sourceforge/kolmafia/session/ForbiddenManagerTest.java

```diff
@@ -0,0 +1,120 @@
+package net.sourceforge.kolmafia.session;
+
+import static internal.helpers.Player.withProperty;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import internal.helpers.Cleanups;
+import net.sourceforge.kolmafia.KoLCharacter;
+import net.sourceforge.kolmafia.preferences.Preferences;
+import net.sourceforge.kolmafia.request.MallPurchaseRequest;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+public class ForbiddenManagerTest {
+  @BeforeAll
+  public static void beforeAll() {
+    KoLCharacter.reset("ForbiddenManager");
```

Comment:
Similar comment as before, minus the reference to spaces.

Reply:
Same response, minus the reference to spaces. :)